elastic/kibana
# [ML][Fleet] Link to ML assets from Integration > Assets tab

## Summary

Related issue: https://github.com/elastic/kibana/issues/182199

This PR adds links for ML assets installed via Integrations. 

For `transform` and `ml_model` (as they are ES asset types) the manual mappings in [get_bulk_assets.ts](https://github.com/elastic/kibana/blob/ac6f643904c6bb7e8342b68e470e933e33a7c206/x-pack/plugins/fleet/server/services/epm/packages/get_bulk_assets.ts#L21) has been modified to include a link to the Transform management page and the Trained models page, respectively. The pages will be filtered to the asset id.

This PR also adds the ability to save state in the url for the Transform list to allow the url to link to a filtered list of transforms.  

### To test:

- From the side navigation - click the `Add integrations` button at the bottom to get to the Integrations page.
<img width="318" alt="image" src="https://github.com/user-attachments/assets/d4632221-7f83-4678-ac0f-cb1e20853a6d">

- To test ml models link, install the `Living off the Land Attack Detection` integration
- To test transform link, install the `Lateral Movement Detection` integration

- Once they are installed you can navigate to the `Installed Integrations` tab on the Integrations page
<img width="660" alt="image" src="https://github.com/user-attachments/assets/8dc1db76-4b93-4057-b502-a90980a2a484">

- Select the installed package you want to view and then go to the `Assets` tab
<img width="1256" alt="image" src="https://github.com/user-attachments/assets/c3382f9d-b1ed-4043-ac3f-73180effefe8">

- You can then expand the desired asset section (Ml models or transforms) and click the link to ensure it takes you to the correct place


### Checklist

Delete any items that are not applicable to this PR.
